Build errors on js-walletconnect-core when publishing

I ran `npm run publish` from root to deploy all packages at once for v0.6.0 but got a build error from `js-walletconnect-core` at `rollup -c rollup.config.js`

<img width="1153" alt="screenshot 2018-09-04 00 26 48" src="https://user-images.githubusercontent.com/10136079/45003326-3c588b80-afd9-11e8-940c-b7eb08eb1a16.png">

Below is the full error stack just after script run that tests successfully 

```bash
lerna success run Ran npm script 'test' in 3 packages:
lerna success - js-walletconnect-core
lerna success - rn-walletconnect-wallet
lerna success - walletconnect

> walletconnect-monorepo@0.6.0 lint /Users/pedrogomes/_walletconnect/walletconnect-monorepo

> eslint packages/*/src/**/*.js packages/*/test/**/*.spec.js


> walletconnect-monorepo@0.6.0 build /Users/pedrogomes/_walletconnect/walletconnect-monorepo
> lerna run build

lerna notice cli v3.1.4
lerna ERR! npm run build exited 1 in 'js-walletconnect-core'
lerna ERR! npm run build stdout:

> js-walletconnect-core@0.6.0 build /Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core
> rollup -c rollup.config.js


lerna ERR! npm run build stderr:

src/index.js → dist/js-walletconnect-core.cjs.js, dist/js-walletconnect-core.esm.js...
[!] (Plugin at pos 2 plugin) SyntaxError: Unexpected token (258:25) in /Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/src/connector.js
src/connector.js (258:25)
SyntaxError: Unexpected token (258:25) in /Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/src/connector.js
    at Parser.pp$4.raise (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2756:13)
    at Parser.pp.unexpected (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:647:8)
    at Parser.pp$3.parseIdent (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2711:10)
    at Parser.pp$3.parsePropertyName (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2495:105)
    at Parser.pp$3.parseProperty (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2425:8)
    at Parser.pp$3.parseObj (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2379:23)
    at Parser.pp$3.parseExprAtom (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2179:17)
    at Parser.pp$3.parseExprSubscripts (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2047:19)
    at Parser.pp$3.parseMaybeUnary (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:2024:17)
    at Parser.pp$3.parseExprOps (/Users/pedrogomes/_walletconnect/walletconnect-monorepo/packages/walletconnect-core/node_modules/acorn/dist/acorn.js:1966:19)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! js-walletconnect-core@0.6.0 build: `rollup -c rollup.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the js-walletconnect-core@0.6.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pedrogomes/.npm/_logs/2018-09-03T23_22_33_957Z-debug.log```

--- a/packages/walletconnect-core/package.json
+++ b/packages/walletconnect-core/package.json
@@ -40,7 +40,7 @@
     "@babel/register": "7.0.0",
     "mocha": "5.2.0",
     "rollup": "0.65.0",
-    "rollup-plugin-babel": "4.0.2",
+    "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-commonjs": "9.1.6",
     "rollup-plugin-json": "3.0.0",
     "rollup-plugin-node-builtins": "2.1.2",

--- a/packages/walletconnect-core/rollup.config.js
+++ b/packages/walletconnect-core/rollup.config.js
@@ -4,7 +4,7 @@ const commonjs = require('rollup-plugin-commonjs')
 const globals = require('rollup-plugin-node-globals')
 // const builtins = require('rollup-plugin-node-builtins')
 const json = require('rollup-plugin-json')
-// const babel = require('rollup-plugin-babel')
+const babel = require('rollup-plugin-babel')
 // const uglify = require('rollup-plugin-uglify')
 
 const pkg = require('./package.json')
@@ -26,13 +26,13 @@ export default [
   //       browser: true,
   //       preferBuiltins: false
   //     }),
+  //     babel({
+  //       exclude: ['node_modules/**'] // only transpile our source code
+  //     }),
   //     commonjs(), // so Rollup can convert `crypto` to an ES module
   //     globals(),
   //     builtins(),
   //     json(),
-  //     babel({
-  //       exclude: ['node_modules/**'] // only transpile our source code
-  //     })
   //     // uglify(),
   //   ]
   // },
@@ -50,6 +50,9 @@ export default [
         modulesOnly: true,
         preferBuiltins: false
       }), // so Rollup can find `crypto`
+      babel({
+        exclude: ['node_modules/**'] // only transpile our source code
+      }),
       commonjs(), // so Rollup can convert `crypto` to an ES module
       globals(),
       json()

--- a/packages/walletconnect-react-native/package.json
+++ b/packages/walletconnect-react-native/package.json
@@ -42,7 +42,7 @@
     "@babel/register": "7.0.0",
     "mocha": "5.2.0",
     "rollup": "0.65.0",
-    "rollup-plugin-babel": "4.0.2",
+    "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-commonjs": "9.1.6",
     "rollup-plugin-json": "3.0.0",
     "rollup-plugin-node-builtins": "2.1.2",

--- a/packages/walletconnect-react-native/rollup.config.js
+++ b/packages/walletconnect-react-native/rollup.config.js
@@ -26,13 +26,13 @@ export default [
         browser: true,
         preferBuiltins: false
       }),
+      babel({
+        exclude: ['node_modules/**'] // only transpile our source code
+      }),
       commonjs(), // so Rollup can convert `crypto` to an ES module
       globals(),
       builtins(),
       json(),
-      babel({
-        exclude: ['node_modules/**'] // only transpile our source code
-      })
       // uglify(),
     ]
   },
@@ -50,6 +50,9 @@ export default [
         modulesOnly: true,
         preferBuiltins: false
       }), // so Rollup can find `crypto`
+      babel({
+        exclude: ['node_modules/**'] // only transpile our source code
+      }),
       commonjs(), // so Rollup can convert `crypto` to an ES module
       globals(),
       json()

--- a/packages/walletconnect/package.json
+++ b/packages/walletconnect/package.json
@@ -41,7 +41,7 @@
     "@babel/register": "7.0.0",
     "mocha": "5.2.0",
     "rollup": "0.65.0",
-    "rollup-plugin-babel": "4.0.2",
+    "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-commonjs": "9.1.6",
     "rollup-plugin-json": "3.0.0",
     "rollup-plugin-node-builtins": "2.1.2",

--- a/packages/walletconnect/rollup.config.js
+++ b/packages/walletconnect/rollup.config.js
@@ -25,13 +25,13 @@ export default [
         browser: true,
         preferBuiltins: false
       }),
+      babel({
+        exclude: ['node_modules/**'] // only transpile our source code
+      }),
       commonjs(), // so Rollup can convert `crypto` to an ES module
       globals(),
       builtins(),
-      json(),
-      babel({
-        exclude: ['node_modules/**'] // only transpile our source code
-      })
+      json()
       // uglify(),
     ]
   },
@@ -48,6 +48,9 @@ export default [
         modulesOnly: true,
         preferBuiltins: false
       }), // so Rollup can find `crypto`
+      babel({
+        exclude: ['node_modules/**'] // only transpile our source code
+      }),
       commonjs(), // so Rollup can convert `crypto` to an ES module
       globals(),
       json()

